internal/driver/commands.go

```diff
@@ -199,6 +199,14 @@ var configHelp = map[string]string{
 		"Use name=value syntax to limit the matching to a specific tag.",
 		"Numeric tag filter examples: 1kb, 1kb:10kb, memory=32mb:",
 		"String tag filter examples: foo, foo.*bar, mytag=foo.*bar"),
+	"tagleaf": helpText(
+		"Adds a pseudo leaf location for samples with a matching tag.",
+		"Use name=value syntax to specify the tag and the value for which nodes would be added.",
+		"Example: latency=1s."),
+	"tagroot": helpText(
+		"Adds a pseudo root node for samples with a matching tag.",
+		"Use name=value syntax to specify the tag and the value for which nodes would be added.",
+		"Example: latency=1s."),
```

Comment:
Is the dot part of the syntax? I assume not which makes this example non copy-pasteable.

Reply:
Dropped dot.